mk/cuda.mk

```diff
@@ -74,14 +74,14 @@ cudnn-runtime: $(CUDNN_LATEST)-runtime
 %-cudnn3-runtime: %-runtime $(CURDIR)/%/runtime/cudnn3/Dockerfile
 	$(NV_DOCKER) build -t cuda:$@ $(CURDIR)/$*/runtime/cudnn3
 
-%-cudnn4-devel: %-devel $(CURDIR)/%/devel/cudnn4/Dockerfile
+%-cudnn4-devel: %-devel $(CURDIR)/$@/devel/cudnn4/Dockerfile
```

Comment:
That was a mistake. I must've accidentally changed that line without noticing. Thanks for catching.